### PR TITLE
Drop in a fresh new haiku

### PR DESCRIPTION
**Pull request by tcd**
Add a fresh new haiku to the collection.

### CODE REVIEW

File: _haikus/freshness.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: freshness
+author: tcd
+---
+
+the farm has freshness
```
Comment (ltagliaferri) [inline _haikus/freshness.md]: Please add a `<br>` tag or two spaces — `  ` — to break the lines on your haiku.

File: _haikus/freshness.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+both on the ground and way up high
```
Comment (ltagliaferri) [inline _haikus/freshness.md]: This line has one too many syllables, consider this change:

```
 both on the ground and up high<br>
```

File: _haikus/freshness.md (hunk 1 above)
Comment (tcd) [reply _haikus/freshness.md]: break tags added!

File: _haikus/freshness.md (hunk 2 above)
Comment (tcd) [reply _haikus/freshness.md]: Ahh, that one slipped by me. I think that change works perfectly.

**Comment by tcd** [comment]
@ltagliaferri is it alright that I made changes in an additional commit or should I squash them?

**Pull request merged**